radio/src/opentx.cpp

```diff
@@ -1822,8 +1822,8 @@ void opentxInit()
   auxSerialInit(g_eeGeneral.auxSerialMode, modelTelemetryProtocol());
 #endif
 
-#if defined(AUX2_SERIAL)
-  aux2SerialInit();
+#if defined(AUX3_SERIAL)
+  aux3SerialInit();
 #endif
 
```

Comment:
Please also include:
#if defined(AUX4_SERIAL)
  aux4SerialInit();
#endif

This is so that Flysky gimbal works